umbra-js/src/classes/Umbra.ts

```diff
@@ -8,7 +8,6 @@ import {
   BigNumber,
```

Comment:
*Arbitrary comment location*

The form doesn't currently default to a token. It should default to the base asset as the main send form does.

Reply:
Fixed in https://github.com/ScopeLift/umbra-protocol/pull/491/commits/7d836b1a1d9b24d50624e191bf1ce1cede053541